pkg/crd/k8c.io/kubermatic.k8c.io_clusters.yaml

```diff
@@ -1945,6 +1945,19 @@ spec:
                 description: HumanReadableName is the cluster name provided by the
                   user.
                 type: string
+              imagePullSecret:
+                description: 'Optional: ImagePullSecret references a secret containing
+                  container registry credentials that are set on node level.'
+                properties:
+                  name:
+                    description: Name is unique within a namespace to reference a
+                      secret resource.
+                    type: string
+                  namespace:
+                    description: Namespace defines the space within which the secret
+                      name must be unique.
```

Comment:
I find this sentence a bit confusing. I mean the k8s api enforces that the name of the secret is unique. What I mean is that you cannot just create another secret with the same name in the same namespace. 
In addition I would want to stress in the description that this is the secret the controller looks for registry credentials

Reply:
okay no. Then it's fine :)